homepage.css

```diff
@@ -116,14 +124,17 @@ ul li:hover ul{
 .content_1 p{
 	font-size:16px;
 	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
 .content_1 h1{
 	font-size:24px;
-	color:black;
+	color:#0C59CF;
```

Comment:
10/09/2021: Changing the fonts style for the content_1(heading of text that talk briefly about the music database) to be Open Sans, also the font weight. Also change it the font color to navy blue.